src/modules/currencyWallets/api.js

```diff
@@ -17,8 +17,8 @@ import {
   setCurrencyWalletTxMetadata,
   setupNewTxMetadata
 } from '../actions.js'
-import { makeShapeshiftApi } from '../exchange/exchangeApi.js'
-import type { ExchangeSwapRate } from '../exchange/exchangeApi.js'
+import makeShapeshiftApi from '../exchange/api.js'
```

Comment:
It's really hard to debug filenames like this, since you're never sure which `api.js` you want when there are several with the same name. A better name would be `shapeshift.js`.